CompanyManager.DAL/Repository/PersonService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompanyManager.DAL.Repository
+{
+    class PersonService
+    {
+        //CRUD functions on Person Table
```

Comment:
Tutaj poczytaj sobie o disconnected scenario trochę, to Ci wyjaśnię jeszcze, ale tutaj wydaje się że taki scenariusz będzie.

https://www.entityframeworktutorial.net/EntityFramework5/attach-disconnected-entity-graph.aspx
https://www.entityframeworktutorial.net/entityframework6/save-disconnected-entity-in-entity-framework.aspx

czyli generalnie będziesz korzystać z using(var context = new PersonContext()){ } i w tym będzie potrzebny kod. Taka konstrukcja powoduje to, że tworzony jest kontekst, później jest robiona operacja zapisana w { } i na końcu kontekst jest czyszczony (dispose). To też jest często na pytaniach rekrutacyjnych co robi using. W razie wątpliwości pytaj to postaram się wyjaśnić. Ten service (usługa) powinna myślę być napisana w scenariuszu disconnected.